app/src/main/java/com/mensinator/app/settings/SettingsScreen.kt

```diff
@@ -443,6 +451,67 @@ private fun ColorPickerPreview() {
     }
 }
 
+@Composable
+private fun SettingText(
+    text: String,
+    modifier: Modifier = Modifier,
+    dbKey: String,
+) {
+    val dbHelper: IPeriodDatabaseHelper = koinInject()
+    val message = dbHelper.getSettingByKey(dbKey)?.value.toString()
+    var showDialog by remember { mutableStateOf(false) }
```

Comment:
Could be added to the `ViewState` but this is a bit of a personal choice. I'm leaning towards adding it there for consistency

Reply:
Please check now and see if I've done it properly

Reply:
Yep